src/main/java/com/bobocode/svydovets/bibernate/util/EntityUtils.java

```diff
@@ -155,4 +155,23 @@ public static <T> void setValueToField(T instance, Field field, Object value) {
                     e);
         }
     }
+
+    public static <T> void updateManagedEntityField(T detachedEntity, T managedEntity, Field field) {
```

Comment:
I think better to rename detachedEntity to toEntity, and managedEntity fromEntity and change the position
like pdateManagedEntityField(T fromEntity, T toEntity, Field field)
to make more generic and descriptive